src/psfmachine/machine.py

```diff
@@ -1127,8 +1129,8 @@ def _update_source_mask_remove_bkg_pixels(self, flux_cut_off=1, frame_index="mea
         #    (self.mean_model.max(axis=1) < 1)
         # )
 
-        # Recreate mean model!
-        self._get_mean_model()
+        # Recreate normalized mean model!
+        self._get_normalized_mean_model()
```

Comment:
Should this be `_renormalize_mean_model()`?

Reply:
Is there a check to make sure the model isn't already normalized?

Reply:
Yes, lines`1193-1195` check if the model is normalized. 
The method computes the factor before creating the final `mean_model`, that's why it is named like that.